TelCo.ColorCoder/Program.cs

```diff
@@ -1,147 +1,150 @@
-            Debug.Assert(testPair1.minorColor == Color.Brown);
-
-            pairNumber = 5;
-            testPair1 = Program.GetColorFromPairNumber(pairNumber);
-            Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
-            Debug.Assert(testPair1.majorColor == Color.White);
-            Debug.Assert(testPair1.minorColor == Color.SlateGray);
-
-            pairNumber = 23;
-            testPair1 = Program.GetColorFromPairNumber(pairNumber);
-            Console.WriteLine("[In]Pair Number: {0},[Out] Colors: {1}\n", pairNumber, testPair1);
-            Debug.Assert(testPair1.majorColor == Color.Violet);
-            Debug.Assert(testPair1.minorColor == Color.Green);
-
-            ColorPair testPair2 = new ColorPair() { majorColor = Color.Yellow, minorColor = Color.Green };
-            pairNumber = Program.GetPairNumberFromColor(testPair2);
-            Console.WriteLine("[In]Colors: {0}, [Out] PairNumber: {1}\n", testPair2, pairNumber);
-            Debug.Assert(pairNumber == 18);
-
-            testPair2 = new ColorPair() { majorColor = Color.Red, minorColor = Color.Blue };
-            pairNumber = Program.GetPairNumberFromColor(testPair2);
-            Console.WriteLine("[In]Colors: {0}, [Out] PairNumber: {1}", testPair2, pairNumber);
-            Debug.Assert(pairNumber == 6);
-        }
-    }
-}
+﻿using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    /// <summary>
+    /// The 25-pair color code, originally known as even-count color code, 
+    /// is a color code used to identify individual conductors in twisted-pair 
+    /// wiring for telecommunications.
+    /// This class provides the color coding and 
+    /// mapping of pair number to color and color to pair number.
+    /// </summary>
+    class Program
+    {
+        /// <summary>
+        /// Array of Major colors
+        /// </summary>
+        public static Color[] colorMapMajor;
+        /// <summary>
+        /// Array of minor colors
+        /// </summary>
+        public static Color[] colorMapMinor;
+        /// <summary>
+        /// data type defined to hold the two colors of clor pair
+        /// </summary>
+        //internal class ColorPair
+        //{
+        //    internal Color majorColor;
+        //    internal Color minorColor;
+        //    public override string ToString()
+        //    {
+        //        return string.Format("MajorColor:{0}, MinorColor:{1}", majorColor.Name, minorColor.Name);
+        //    }
+        //}
+        /// <summary>
+        /// Static constructor required to initialize static variable
+        /// </summary>
+        static Program()
+        {
+            colorMapMajor = new Color[] { Color.White, Color.Red, Color.Black, Color.Yellow, Color.Violet };
+            colorMapMinor = new Color[] { Color.Blue, Color.Orange, Color.Green, Color.Brown, Color.SlateGray };
+        }
+
+        /// <summary>

```

Comment:
Never leave commented code in commits. Remove this.

Reply:
Removed few comments left only the descriptive comments..